elements/nuxeo-csv-export/nuxeo-csv-export-button.html

```diff
@@ -78,35 +81,41 @@
           type: Boolean,
           value: false,
         },
+
+        /**
+         * Current action status.
+         */
+        status: {
+          type: Object,
+          notify: true,
+        },
+
         _label: {
           type: String,
           computed: '_computeLabel(i18n)',
-        },
-        _lastStatus: Object
+        }
       },
 
       execute: function() {
-        this.$.resource.path = this.provider.$.nxResource.path.replace(/\/execute$/, '/bulk/csvExport');
-        this.$.resource.params = this.provider.$.nxResource.params;
-        this.$.resource.data = {};
-        var schemas = this.schemas != null ? this.schemas : this.provider.$.nxResource.schemas;
+        var actionParams = {};
+        var schemas = this.schemas ? this.schemas : this.provider.schemas;
```

Comment:
This won't allow the `schemas` property to be an empty string. We should check for `undefined` and `null`:
```suggestion
        var schemas = this.schemas != null ? this.schemas : this.provider.schemas;
```

Reply:
Ok, fair enough, done 👍